### ISSUE TITLE
Add data collection method

### ISSUE THREAD

**Issue by waynegraham** [waynegraham/hidden_collections_registry#18]
We need a method to collect data from other collections. 

- [ ] Create a Google Form
- [ ] Add a link to the form on the site 
- [ ] Add backend processing for interacting with Google API

**Comment by nkferraiolo** [comment]
I can work on the Google Form

**Comment by waynegraham** [comment]
Does this need to go by Friday?

**Comment by nkferraiolo** [comment]
No. I've put in the blurb that people can email us to submit collections. This will buy us some time to get the form in place.
